README.md

```diff
@@ -2,6 +2,15 @@
 
 ## Create a .env.local file in each service with the following keys:
 - PORT
+- SLACK_WEBHOOK
+- SLACK_BOT_USERNAME
+- SLACK_BOT_PROFILE_IMAGE_URL
+- ERROR_SLACK_BOT_USERNAME
+- ERROR_ICON_EMOJI
```

Comment:
Let's create `.env.sample` file to keep these keys

And we can mention in `README` to copy this file
```bash
cp .env.sample .env.local
```

Reply:
Sure